spec/controllers/sections_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe SectionsController, type: :controller do
+
+  describe "sections#new action" do
+    it "should display the new section page" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "sections#create action" do
+    it "should create a new section in the database" do
+      post :create, section: {name: 'Availabilities'}
+      expect(response).to redirect_to root_path
+
+      section = Section.last
+      expect(section.name).to eq("Availabilities")
+    end
+  end
+
+  # describe "sections#show action" do
+  #   it "should display a section to a user" do
+  #     new_section = Section.create(name: 'Availabilities')
+  #     get :show, new_section.id
```

Comment:
Gotcha. Will fix that now.

On Fri, Mar 25, 2016 at 4:19 PM, ken mazaika <notifications@github.com>
wrote:

> In spec/controllers/sections_controller_spec.rb
> <https://github.com/FirehoseCommunity/Lets-Pair-Program/pull/28#discussion_r57501691>
> :
>
> > +  end
> > +
> > +  describe "sections#create action" do
> > +    it "should create a new section in the database" do
> > +      post :create, section: {name: 'Availabilities'}
> > +      expect(response).to redirect_to root_path
> > +
> > +      section = Section.last
> > +      expect(section.name).to eq("Availabilities")
> > +    end
> > +  end
> > +
> > +  # describe "sections#show action" do
> > +  #   it "should display a section to a user" do
> > +  #     new_section = Section.create(name: 'Availabilities')
> > +  #     get :show, new_section.id
>
> should be
>
> get :show, id: new_section.id
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/FirehoseCommunity/Lets-Pair-Program/pull/28/files/51650d2257c57b02544fbfb63e34f108be1359ed#r57501691>
>